docs/agents/sematext-agent/agent-troubleshooting.md

```diff
@@ -0,0 +1,31 @@
+title: Agent Troubleshooting
+
+If you are experiencing issues with the Sematext Agent, there are some things you can try.
+
+1. Check the Sematext Agent logs and filter by errors. By default they'll be located at `/opt/spm/spm-monitor/logs/st-agent`. 
+   1. If you see errors related to permission denied while accessing log files or system information, stop the agent and start it again with elevated privileges. 
+   2. If you see errors like `"cloud" collector failed to complete the initial run` check out our [tags common schema docs](https://sematext.com/docs/tags/common-schema).
+   
+
+2. Try restarting the agent. Specific instructions on how to do so depending on your environment can be found [here](https://sematext.com/docs/agents/sematext-agent/starting-stopping).
+
+3. Update the agent to the latest version. For more details on how to do that, see our [Agent Upgrade Instructions](https://sematext.com/docs/monitoring/spm-faq/#agent-updating).
+
+4. Journal is a buffer we use to store failed events. In case of outages or internal communication issues, the Journal files may grow rapidly and produce high system load. If you are experiencing high CPU usage, do the following:
+   1. Navigate to the journal directory, by default `/opt/spm/spm-monitor/st-agent/journal`.
+   2. See if it contains a large number of files. If it does, it might be the cause of high CPU usage. You can disable Journal continuing to follow these steps.
```

Comment:
You can disable Journal continuing to follow these steps. ==> You can disable Journal by following these steps.

Reply:
I changed the structure of the bullet points a little bit as well as the wording